packages/@webex/common/test/unit/spec/browser-detection.js

```diff
@@ -0,0 +1,37 @@
+/*!
+ * Copyright (c) 2015-2020 Cisco Systems, Inc. See LICENSE file.
+ */
+
+import {assert} from '@webex/test-helper-chai';
+import sinon from 'sinon';
+import { getBrowserSerial } from '@webex/common';
+import window from 'global/window';
+import bowser from 'bowser';
+import {browserDetection} from '@webex/common/src/constants.js';
+
+describe('getBowserSerial()', () => {
+  it('should provide the parsed bowser user agent when available', () => {
+    //@ts-ignore
+    window.navigator = {userAgent: 'user agent data'};
```

Comment:
For test scope safety, one small thing we can do is:

```js
describe('getBrowserSerial()', () => {
  const originalWindowNavigator = {...window.navigator}; // allows us to reset this after the last test
  
  after('restore window.navigator', () => {
    window.navigator = originalWindowNavigator; // reset AFTER this describe
  });
  
  /* ...the rest of your test... */
});
```

This will prevent any contamination in which `window.navigator` is checked outside of this test but within the same runner.